make.py

```diff
@@ -114,11 +117,16 @@ def translate(self, text):
 
 
 class BEPUB:
-    def __init__(self, epub_name, model, key):
+    def __init__(self, epub_name, model, key, resume):
         self.epub_name = epub_name
         self.new_epub = epub.EpubBook()
         self.translate_model = model(key)
         self.origin_book = epub.read_epub(self.epub_name)
+        self.p_t = []
```

Comment:
let's make the name more reasonable.
My to do is just a hint.

like self.p_to_save or you can choose a better name